### PR TITLE
Note about devtools optional permission

### PR DESCRIPTION
**Pull request by rebloor**
See https://github.com/mdn/content/pull/27282

### CODE REVIEW

File: src/content/documentation/develop/request-the-right-permissions.md [hunk 1]
```diff
@@ -102,7 +102,9 @@ Not all [permissions](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExte
...
 | `clipboardRead`                                                                                                                | Get data from the clipboard                                                                                                                                                                                      |
 | Other permissions:                                                                                                               |
-| The manifest key [`devtools_page`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/devtools_page) | Extend developer tools to access your data in open tabs                                                                                                                                                          |
+| The manifest key [`devtools_page`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/devtools_page)<sup>*<sup> | Extend developer tools to access your data in open tabs                                                                                                                                                          |
```
Comment (Rob--W) [inline src/content/documentation/develop/request-the-right-permissions.md:105]:
```suggestion
| The manifest key [`devtools_page`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/devtools_page)<sup>*</sup> | Extend developer tools to access your data in open tabs                                                                                                                                                          |
```

Insert `/` in second `<sup>`

Comment (Rob--W) [reply src/content/documentation/develop/request-the-right-permissions.md:105]: Not sure if one asterisk is sufficiently obvious. Would a few extra words between the current text and the asterisk improve clarity? I.e. ending up with `The manifest key devtools_page or the "devtools" optional permission`.

**Review by Rob--W — COMMENTED** [review]
Not sure whether

**Comment by rebloor** [comment]
@Rob--W as the note follows the table line, it works for me:
image

**Review by Rob--W — APPROVED** [review]

**Pull request merged**